### PR TITLE
Oni Horn Markings

### PR DESCRIPTION
**Pull request by dootythefrooty**
# Description

---

Three new Oni horn markings,] two of which have three tone support. There was a distinct lack of large horns, which I tried my hand at.

<details><summary><h1>Media</h1></summary>
<p>

display

</p>
</details>

---

# Changelog

:cl:
- add: Added crowned, tall bull, and tall curved Oni horns.

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+- type: marking
+  id: OniHornTallCurved
+  bodyPart: HeadTop
+  markingCategory: HeadTop
+  forcedColoring: false
+  speciesRestriction: [Oni]
+  sprites:
+  - sprite: Mobs/Customization/oni/oni_horns48x48.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml:8]:
```suggestion
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
```

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    state: tall_curved_3tone_1
+  - sprite: Mobs/Customization/oni/oni_horns48x48.rsi
+    state: tall_curved_3tone_2
+  - sprite: Mobs/Customization/oni/oni_horns48x48.rsi
+    state: tall_curved_3tone_3
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml:22]:
```suggestion
  sprites:
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
    state: tall_curved_3tone_1
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
    state: tall_curved_3tone_2
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
    state: tall_curved_3tone_3
```

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- type: marking
+  id: OniHornTallBull
+  bodyPart: HeadTop
+  markingCategory: HeadTop
+  forcedColoring: false
+  speciesRestriction: [Oni]
+  sprites:
+  - sprite: Mobs/Customization/oni/oni_horns48x48.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml:30]:
```suggestion
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
```

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    state: tall_bull_3tone_1
+  - sprite: Mobs/Customization/oni/oni_horns48x48.rsi
+    state: tall_bull_3tone_2
+  - sprite: Mobs/Customization/oni/oni_horns48x48.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml:43]:
```suggestion
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
    state: tall_bull_3tone_1
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
    state: tall_bull_3tone_2
  - sprite: Mobs/Customization/Oni/oni_horns48x48.rsi
```

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    state: tall_bull_3tone_3
+- type: marking
+  id: OniHornCrowned
+  bodyPart: HeadTop
+  markingCategory: HeadTop
+  forcedColoring: false
+  speciesRestriction: [Oni]
+  sprites:
+  - sprite: Mobs/Customization/oni/oni_horns.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/oni_horns.yml:52]:
```suggestion
  - sprite: Mobs/Customization/Oni/oni_horns.rsi
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
Test fail valid

**Review by VMSolidus — APPROVED** [review]

**Review by DangerRevolution — APPROVED** [review]

**Pull request merged**